spec/request/service_instances_spec.rb

```diff
@@ -2304,6 +2332,29 @@ def expect_metadata(instance, annotations: [], labels: [])
     expect(l).to match_array(labels)
   end
 
+  def share_service_instance(instance, target_space)
+    enable_sharing!
+
+    share_request = {
+      'data' => [
+        { 'guid' => target_space.guid }
+      ]
+    }
+
+    post "/v3/service_instances/#{instance.guid}/relationships/shared_spaces", share_request.to_json, admin_headers
+    expect(last_response.status).to eq(200)
+  end
+
+  def enable_sharing!
+    feature_flag = VCAP::CloudController::FeatureFlag.where(name: 'service_instance_sharing').first
+    if feature_flag.nil?
+      VCAP::CloudController::FeatureFlag.make(name: 'service_instance_sharing', enabled: true)
+    else
+      feature_flag.enabled = true
+      feature_flag.save
+    end
```

Comment:
```suggestion
    VCAP::CloudController::FeatureFlag.
      find_or_create(name: 'service_instance_sharing') { |ff| ff.enabled = true }.
      update(enabled: true)
```

This was a gratuitous reason to stretch the Rubydoc muscle and see what I can remember about how the language works.  Feel free to ignore this one.

Reply:
No that's great :D